Dockerfile-e2e

```diff
@@ -2,16 +2,15 @@ FROM 113355358853.dkr.ecr.eu-west-1.amazonaws.com/ho/hmpo/lev/fro/cypress-testin
 # Use image below for local testing. Proxies also require removal.
 #FROM cypress/included:cypress-12.17.1-node-18.16.0-chrome-114.0.5735.133-1-ff-114.0.2-edge-114.0.1823.51-1
 
-ENV HTTP_PROXY='http://proxy.local:8080'
-ENV HTTPS_PROXY='http://proxy.local:8080'
+ARG HTTP_PROXY=''
+ARG HTTPS_PROXY=''
 
 WORKDIR /app
 COPY ./ /app
 
 RUN npm ci -D
 
-ENV HTTP_PROXY=''
-ENV HTTPS_PROXY=''
+ARG NO_PROXY=''
```

Comment:
If this needs to be set, then just a very minor thing, but could you keep the `ARG`s next to each-other?

Reply:
Interestingly this isn't set in https://github.com/UKHomeOffice/lev-api/pull/145, is this required?

Reply:
I have removed